### DISCUSSION TITLE
Awesome!

### DISCUSSION THREAD

**Discussion by kylegl** [Colonel-Sandvich/trpc-vue-query#5] [General]
Hi,

Glad to see this repo. I've been hacking this together for a while. I've attempted this a few times but could never get the types right. Just curious are you using the trpc Next tanstack integration as inspo? If theres anything I can help with lmk.

ps can you enable discussions on this repo?

**Comment by Colonel-Sandvich** [comment]
Hi!

The types are an absolute pain!

I mainly used the [@trpc/react-query](https://github.com/trpc/trpc/tree/main/packages/react-query) package as the main inspiration as that seemed to have the most features and seemed the most mature. But `trpc-nuxt` was definitely the kickstarter for all of this.

I'll take all the help I can get but for the minute I think the best thing you could do is to clone the repo and see if you can follow the quickstart to get the example working. Then tinker with the example until something breaks and make an issue.

If you have any other general advice on open source or tool recommendations I'm all ears.

I will enable discussions now

Thank you

**Comment by kylegl** [comment]
Cool, I'll poke around! You can move this issue to discussions or I can close it.
